lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -230,6 +238,18 @@ def _set_sgi_interface_ingress_flows(self):
             actions = "output:LOCAL"
             self._install_flow(flows.MEDIUM_PRIORITY + 1, match, actions)
 
+        if_addrs = netifaces.ifaddresses(self.config.uplink_bridge).get(netifaces.AF_INET6, [])
+        for addr in if_addrs:
+            addr = addr['addr']
+            # get rid of dev name if present.
+            addr = addr.split('%')[0]
```

Comment:
should we add `if % in addr` check?

Reply:
no need, since we are looking at first element of array any ways.